draft-yasskin-http-origin-signed-responses.md

```diff
@@ -705,7 +705,7 @@ all.
 
 The `Accept-Signature` header field is a Structured Header as defined by
 {{!I-D.ietf-httpbis-header-structure}}. Its value MUST be a parameterised list
-(Section 3.3 of {{!I-D.ietf-httpbis-header-structure}}). Its ABNF is:
+(Section 3.4 of {{!I-D.ietf-httpbis-header-structure}}). Its ABNF is:
 
     Accept-Signature = sh-param-list
 
```

Comment:
Ditto the next two paragraphs. Not sure if you want to stick "primary" in front of all of them, because too verbose. Maybe the first instance per para.